README.md

```diff
@@ -1455,11 +1458,13 @@ Lifecycle Event >>>  SHUTDOWN
 
 You can add event listeners to the distributed data structures.
 
-#### 7.5.2.1. Listening for Map Events
+#### 7.5.2.1. Map Listener
+
+The Map Listener is used by the Hazelcast `Map`.
 
 You can listen to map-wide or entry-based events. To listen to these events, you need to implement the relevant interfaces.
 
-An entry-based  event is fired after the operations that affect a specific entry. For example, `Map.put()`, `Map.remove()` or `Map.evict()`. An `EntryEvent` object is passed to the listener function.
+An entry-based  event is fired after the operations that affect a specific entry. For example, `Map.Put()`, `Map.Remove()` or `Map.Evict()`. An `EntryEvent` object is passed to the listener function.
 
```

Comment:
You may add here an explanation for the following list. 

`You can use the following listeners to listen to entry-based events.`